### PR TITLE
Timer in showcase pages

### PR DESCRIPTION
**Pull request by walaura**
## Why are you doing this?
Add a sense of urgency to the checkout pages for the flash sale. 

Do not merge! this might be an AB test (need to see if we have the audience) and otherwise, it shouldn't go out until Monday

Related: #1069, #1062

[**Trello Card**](https://trello.com/c/OPrDKDBQ/2027-countdown-timer-for-flash-sales)

## Screenshots

screenshot 2018-10-24 at 2 13 34 pm
screenshot 2018-10-24 at 2 52 02 pm

### CODE REVIEW

**Comment by LATaylor-guardian** [comment]
👍

File: assets/helpers/flashSale.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ function getIntcmp(product: SubscriptionProduct, intcmp: ?string, defaultIntcmp:
   return intcmp || defaultIntcmp;
 }
 
-function getEndTime(): number { return new Date(2018, 9, 24, 14, 30, 0).getTime(); }
+function getEndTime(): number { return new Date(2018, 10, 4, 0, 0, 0).getTime(); }
```
Comment (walaura) [inline assets/helpers/flashSale.js]: can we triple check this is correct?

File: assets/components/countdown/countdown.jsx [hunk 2]
```diff
@@ -71,17 +78,29 @@ export default class Countdown extends Component<PropTypes, StateTypes> {
...
     return (
       <time className="component-countdown">
-        {units.map((unit, index) => (
-          <span>
-            <span className="component-countdown__time">{addLeadingZeros(unit, 2)}</span>
-            {index < units.length - 1 && ':'}
+        {days > 0 && chip('days', days)}
+        {chip('hrs', hours)}
+        {chip('mins', minutes)}
+        {chip('secs', seconds)}
```
Comment (walaura) [inline assets/components/countdown/countdown.jsx]: In terms of a11y, I checked the [GDS guidance](https://accessibility.blog.gov.uk/2017/02/08/advice-for-creating-content-that-works-well-with-screen-readers/) and they actually encourage _against_ using any type of additional markup for this, just roll with the awkward spellings.

File: assets/helpers/flashSale.js (hunk 1 above)
Comment (rupertbates) [inline assets/helpers/flashSale.js]: Can you use the same date as is used in the `flashSaleIsActive()` function above? Then we'll only need to change the date in one place next time we do this.

Comment (walaura) [reply assets/helpers/flashSale.js]: Great point!

File: assets/helpers/flashSale.js [hunk 3]
```diff
@@ -6,13 +6,16 @@ import { getProductPrice } from 'helpers/subscriptions';
 
 import type { SubscriptionProduct } from './subscriptions';
 
+// Get end time
+function getEndTime(): number { return new Date(2018, 10, 5, 0, 0).getTime();; }
```
Comment (rupertbates) [inline assets/helpers/flashSale.js]: Can you use the same date as is used in the flashSaleIsActive() function above? Then we'll only need to change the date in one place next time we do this.

Comment (walaura) [reply assets/helpers/flashSale.js]: Great point! all set

**Comment by walaura** [comment]
sorry about the weirdness of `63c4899` right at the end, i was messing with my local clock for testing and as github sorting knows it's set 0 days into the future

**Pull request merged**